File: src/BukkitPE/level/sound/ButtonClickSound.php
<?php
namespace BukkitPE\level\sound;

use BukkitPE\math\Vector3;
use BukkitPE\network\protocol\LevelEventPacket;

class ButtonClickSound extends GenericSound{
    public function __construct(Vector3 $pos, $pitch = 1000){
        parent::__construct($pos, LevelEventPacket::EVENT_SOUND_BUTTON_CLICK, $pitch);
    }
}

File: src/net/BukkitPE/level/sound/ButtonClickSound.java
package net.BukkitPE.level.sound;

import net.BukkitPE.math.Vector3;
import net.BukkitPE.network.protocol.LevelEventPacket;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public class ButtonClickSound extends GenericSound {
    public ButtonClickSound(Vector3 pos) {
        this(pos, 0);
    }

    public ButtonClickSound(Vector3 pos, float pitch) {
        super(pos, LevelEventPacket.EVENT_SOUND_BUTTON_CLICK, pitch);
    }
}
